src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/pydata-sphinx-theme-custom.css

```diff
@@ -715,8 +715,21 @@ button.btn.version-switcher__button:hover {
 .bd-main .bd-content .bd-article-container {
   padding: 8px;
   width: 1000px;
+  max-width: 100%;
 }
 
 .bd-sidebar-primary {
   width: 20%;
 }
+
+@media (min-width: 960px) {
+  .col-lg-3 {
+    width: fit-content;
+  }
+}
+
+@media (min-width: 960px) {
+  .bd-page-width {
+    max-width: min(100%, 1600px) !important;
```

Comment:
@SMoraisAnsys, @RobPasMue increase this size to 1700px will increase the screen size, since 1920 is the standard resolution , i can increase this if you prefer (if you are greedy)

Reply:
![image](https://github.com/user-attachments/assets/0c6bc30e-9570-4ac2-8e8d-4ce0333fed35)

Reply:
The result is great ! Is there a reason for the middle part to be that far away from the scroll bar ?
As far as I like this, I don't have enough hindsight to assess the extent to which this could cause problems (which @RobPasMue  pointed out).

Reply:
> The result is great ! Is there a reason for the middle part to be that far away from the scroll bar ? As far as I like this, I don't have enough hindsight to assess the extent to which this could cause problems (which @RobPasMue pointed out).

good catch, i reduced the gap
![image](https://github.com/user-attachments/assets/44b24452-dbaf-473e-ba67-1a0424cee72c)

Reply:
I think that is going to far.... you probably dont want to have so much space between el central content and the right bar, and so little space between the central content and the scrollbar.....

Reply:
The left and right content is having the same margin and padding, and the scroll bar appears only if there is an overflow in the content. 
![image](https://github.com/user-attachments/assets/89755c99-00e4-44ae-9431-e2ae3c906607)
![image](https://github.com/user-attachments/assets/1a567c91-7992-4464-92fc-690258f62fa7)
These are the 2 changes with and without scrollbar. pinging @mia-guo-ux also here for insights

Reply:
Thanks for pinning me. When the text exceeds the container, I suggest truncating the text with "..." and show the full content upon hovering instead of enabling horizontal scrolling. The vertical scrollbar looks good to me. Also...I agree with the comments that balance is the key :D

Reply:
Thank you all, Then i will go with the above specifications, and about horizontal scrollbar, i will open another PR to fix.